src/sc/contentrules/localrole/config.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+PROJECTNAME = 'sc.contentrules.localrole'
+
+DEFAULT_ENABLED_GROUPS = ('Reviewers',)
```

Comment:
At first I enabled just **Reviewers** group, need to check what makes sense in the use case.